src/infrastructure/services/creditCard/CreditCardService.ts

```diff
@@ -89,5 +121,6 @@ export const {
       matchRejected: getCreditCardByIdRejected,
     },
     deleteCreditCard: { matchFulfilled: deleteCreditCardFulfiled },
+    getPublicKeys: { matchFulfilled: getPublicKeysFulfilled },
```

Comment:
same here as matchers are only used for request that interact with state.

Reply:
Done!